net-irc/atheme-services/files/atheme-services.initd

```diff
@@ -0,0 +1,36 @@
+#!/sbin/openrc-run
+# Copyright 1999-2011 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+extra_started_commands="reload"
+
+depend() {
+	need net
+	use ircd
+	provide irc-services
+}
+
+start() {
+	ATHEME_SERVICES_RUNDIR=/var/run/atheme
+	if ! [ -d "${ATHEME_SERVICES_RUNDIR}" ]; then
+	    ebegin "Creating \`${ATHEME_SERVICES_RUNDIR}'"
+	    install -o atheme-services -d "${ATHEME_SERVICES_RUNDIR}"
+	    eend $?
+	fi
+
+	ebegin "Starting ${SVCNAME}"
+	start-stop-daemon --start --pidfile=/var/run/atheme/atheme.pid --user atheme-services --exec /usr/bin/atheme-services
```

Comment:
Also, given the 'standardness' of this init.d file, I think you could look into replacing the custom functions with `start_pre` combined with the declarative style, i.e. defining `command=`, `command_user=` and `pidfile=`. Look at `man openrc-run`.